src/main/java/com/aerospike/mapper/tools/converters/MappingConverter.java

```diff
@@ -166,7 +166,7 @@ public Object translateToAerospike(Object obj) {
      * Given an instance of a class (of any type), convert its properties to a map, properties names will use as the
      * key and properties values will be the values.
      * @param instance The instance of a class (of any type).
-     * @return a Map of the properties <propertyName, propertyValue> of the given instance.
+     * @return a Map of the properties propertyName, propertyValue of the given instance.
```

Comment:
Simplify it to `the properties {@link Map} of the given instance`. You have all the details in the description above.